decidim_app-design/app/views/admin/partials/_logs_list.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">Admin logs</h2>
+  </div>
+  <div class="table-scroll">
+    <ul id="logs" class="table">
+      <% 5.times do |index| %>
+        <li class="logs-log">
+          <div class="logs-log__content">
+            <div class="logs-log__date">10/10/18 08:24</div>
+            <div class="logs-log__explanation">
+              <a class="logs-log__author">Matthew Mills </a>
+              created
+              <a class="logs-log__resource">Regular meetup</a>
+              in
+              <a class="logs-log__space">Book Cycle</a>
+            </div>
+            <div class="logs-log__actions">
+            </div>
+          </div>
+        </li>
+
+        <li class="logs-log deletion">
```

Comment:
Isn't `deletion` a modifier? If so, according to BEM, this should be `logs-log--deletion`